nixos/modules/services/misc/open-webui.nix

```diff
@@ -0,0 +1,89 @@
+{ config
+, lib
+, pkgs
+, ...
```

Comment:
I think we should avoid `...` in `nixpkgs`.

Reply:
I see it being used in other services too: [ollama](https://github.com/NixOS/nixpkgs/blob/95b54cc906c5daab07fa5e730505b447265751c1/nixos/modules/services/misc/ollama.nix#L1), [ananicy](https://github.com/NixOS/nixpkgs/blob/95b54cc906c5daab07fa5e730505b447265751c1/nixos/modules/services/misc/ananicy.nix#L1). If I don’t I get `error: function 'anonymous lambda' called with unexpected argument…`